cypress/support/pages/home.pageObject.js

```diff
@@ -3,8 +3,23 @@ import PageObject from '../PageObject';
 class homePageObject extends PageObject {
   url = '/#/';
 
-  get usernameLink() {
-    return cy.getByDataCy('/profile/');
+    usernameLink(username) {
```

Comment:
Method names should be more descriptive and follow camelCase naming convention. Consider renaming 'usernameLink' to 'verifyUsernameLink'